torch/utils/data/dataloader.py

```diff
@@ -90,6 +93,18 @@ def __iter__(self):
             yield None
 
 
+def _sharding_worker_init_fn(worker_init_fn, worker_id):
+    global_worker_id = worker_id
+    info = torch.utils.data.get_worker_info()
+    total_workers = info.num_workers
+    datapipe = info.dataset
+    if dist.is_available() and c10d.is_initialized():
```

Comment:
nit: Why not use `dist.is_initialized()`?

Reply:
Because I had no idea it is exists =)